src/components/defectReport/DefectReportCreateForm.tsx

```diff
@@ -0,0 +1,309 @@
+          contentBlocks: [{ body: defectReportNewData.body, title: defectReportNewData.title }],
+          addresses: [
+            {
+              geoLocation: {
+                latitude: selectedPosition?.latitude,
+                longitude: selectedPosition?.longitude
+              }
+            }
+          ],
+          mediaContents: [
+            {
+              sourceUrl: {
+                url: imageUrl
+              },
+              contentType: 'image'
+            }
+          ]
+        }
+      });
+
+      navigation.goBack();
+      Alert.alert(texts.defectReport.successScreen.header, texts.defectReport.successScreen.entry);
+    } catch (error) {
+      setIsLoading(false);
+
+      Alert.alert(texts.defectReport.alerts.hint, texts.defectReport.alerts.error);
+    }
+  };
+
+  return (
+    <>
+      <Wrapper style={styles.noPaddingTop}>
+        <Controller
+          name="categoryName"
+          render={({ name, onChange, value }) => (
+            <DropdownInput
+              {...{
+                errors,
+                data: categoryNameDropdownData,
+                value,
+                valueKey: 'name',
+                onChange,
+                name,
+                required: true,
+                label: texts.defectReport.categoryName,
+                placeholder: texts.defectReport.categoryName,
+                control
+              }}
+            />
+          )}
+          control={control}
+        />
+      </Wrapper>
+
+      <Wrapper style={styles.noPaddingTop}>
+        <Input
+          name="title"
+          label={`${texts.defectReport.inputTitle} *`}
+          placeholder={texts.defectReport.inputTitle}
+          validate
+          rules={{
+            required: `${texts.defectReport.inputTitle} ${texts.defectReport.inputErrorText}`
+          }}
+          errorMessage={errors.title && errors.title.message}
+          control={control}
+        />
+      </Wrapper>
+
+      <Wrapper style={styles.noPaddingTop}>
+        <Input
+          name="body"
+          label={`${texts.defectReport.inputDescription} *`}
+          placeholder={texts.defectReport.inputDescription}
+          validate
+          multiline
+          rules={{
+            required: `${texts.defectReport.inputDescription} ${texts.defectReport.inputErrorText}`
+          }}
+          errorMessage={errors.body && errors.body.message}
+          control={control}
+        />
+      </Wrapper>
+
+      <Wrapper style={styles.noPaddingTop}>
+        <Controller
+          name="images"
+          render={(field) => (
+            <ImageSelector
+              {...{
+                control,
+                field,
+                item: {
+                  name: 'images',
+                  label: texts.volunteer.images,
+                  buttonTitle: texts.volunteer.addImage
+                }
+              }}
+            />
+          )}
+          control={control}
+        />
+      </Wrapper>
```

Comment:
After selecting an image, I get a screen like this. The delete image button is not visible on the screen and is not centred exactly.

![IMG_D5142462639D-1](https://user-images.githubusercontent.com/11755668/207922894-35fb29e7-b07e-47b8-b9c1-d37181f9906f.jpeg)

Reply:
ok nice catch, we need a condition for wider screens